include/alpaca/detail/type_info.h

```diff
@@ -1,11 +1,16 @@
 #pragma once
 #include <alpaca/detail/field_type.h>
+#include <alpaca/detail/is_bitset.h>
 #include <alpaca/detail/is_specialization.h>
 
 #ifndef ALPACA_EXCLUDE_SUPPORT_STD_ARRAY
 #include <array>
 #endif
 
+#ifndef ALPACA_EXCLUDE_SUPPORT_STD_BITSET
+#include <bitset>
```

Comment:
should we only include the `is_bitset.h` here under the guard?

Reply:
Good point. Either only here or wherever it is included also add the guard to make sure it doesn't get pulled in unnecessarily.